starfish/stack.py

```diff
@@ -553,19 +553,32 @@ def apply(
                 func, is_volume=is_volume, in_place=True, verbose=verbose, **kwargs
             )
 
-        mapfunc: Callable = map  # TODO: ambrosejcarr posix-compliant multiprocessing
-        indices = list(self._iter_indices(is_volume=is_volume))
+        if is_volume:
+            self._data = self._data.stack(tiles=[Indices.ROUND.value,
+                                                 Indices.CH.value])
+        else:
+            self._data = self._data.stack(tiles=[Indices.ROUND.value,
+                                                 Indices.CH.value,
+                                                 Indices.Z.value])
 
         if verbose:
```

Comment:
idk how you feel about one-liners, but if we do my above suggestion, this could be: 

`tiles = tqdm(self._data.tiles) if verbose else self._data.tiles`

Reply:
love it.